examples/esm-http-ts/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
+    /* Language and Environment */
+    "target": "ES2020" /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */,
```

Comment:
is this compatible with node.js 14 mentioned as minimum version in package.json?

Reply:
The unit tests pass for Node 14. This is an example app and I chose an in-between target as opposed to the oldest or newest because it seemed more realistic that an end user might have in their application.